cluster/swarm/cluster.go

```diff
@@ -968,6 +968,31 @@ func (c *Cluster) BuildImage(buildContext io.Reader, buildImage *types.ImageBuil
 	return nil
 }
 
+// RefreshEngines refreshes all containers in the cluster (UCP-specific)
```

Comment:
@wsong is this useful in any other setting? If so, I don't think we should mention UCP or other closed-source software here.